demo-project/.gitignore

```diff
@@ -23,6 +23,9 @@ logs/**
 # also keep all .gitkeep files
 !.gitkeep
 
+# logging.yml 
+conf/base/logging.yml
+
```

Comment:
I don't think this should be git ignored. It's actually probably a good idea to commit the debug level change.